runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -559,12 +563,15 @@ std::vector<std::shared_ptr<exec::ExecutorMap>> Compiler::compile(const char *pa
   //       execution between interpreter and compiled executor (including control flow)
   if (_options.disable_compile)
   {
+    std::vector<std::shared_ptr<CompilerArtifact>> results;
```

Comment:
@chunseoklee PTAL. As I understand, you are going to introduce some class for collection of `ExecutorMap`.

Reply:
Not decided yet. I will take care of it again after this PR is merged. At this time, this looks fine so far.